src/Lykke.AzureStorage/Tables/AzureTableStorage.cs

```diff
@@ -963,6 +963,19 @@ public Task ScanDataAsync(TableQuery<T> rangeQuery, Func<IEnumerable<T>, Task> c
             return result.Take(n);
         }
 
+        public virtual async Task<T> GetTopRecordAsync(TableQuery<T> query)
```

Comment:
Will be good to add overload with number of records to get and these new methods from overloads which accepts partition.